src/components/UserManagement/UserTableData.jsx

```diff
@@ -474,7 +475,7 @@ const UserTableDataComponent = (props) => {
       <td>
         {!isCurrentUser && (
           <>
-            {!canChangeUserStatus ? (
+            {!canSetFinalDay ? (
```

Comment:
```suggestion
            {canChangeUserStatus || canSetFinalDay ? (
```
or
```suggestion
            {canSetFinalDay ? (
```

1. It looks like the logic for this is inverted? (with the '!')
2. Should `changeUserStatus` also be controlling this, or should it be fully replaced?

Reply:
My apologies, I think I misread what was in the parentheses. You were right. My bad.